ec/dreamcoder/domains/arc/arcPrimitivesIC2.py

```diff
@@ -25,6 +25,10 @@
 
 tcount = baseType("count")
 Count = NewType("Count", int)
+
+SOLVERS_PROGRAM_PATH = os.path.join(os.path.dirname(__file__), "../../../", "solvers/program.ml")
+AUTO_GEN_START = '(* AUTO GEN COMMENT START *)'
```

Comment:
Making an assumption that this is what's intended but would like confirmation (these are OCAML comment format)